assets/scss/_capability_tables.scss

```diff
@@ -44,45 +45,6 @@
     }
   }
 
-  // scroll shadows to indicate more content
```

Comment:
The shadows did not behave properly: they applied to non-scrollable tables, and did not remain at the edge, I think it's better to get rid of these for now:

<img width="295" height="217" alt="Image" src="https://github.com/user-attachments/assets/52975dc9-addc-4a5a-a32d-2446a1a5f450" />

<img width="439" height="184" alt="Image" src="https://github.com/user-attachments/assets/91f9ea32-c37a-4cb4-bd8f-64755f8ad6c4" />